docs/contributing/resoto.md

```diff
@@ -0,0 +1,103 @@
+---
+sidebar_label: Resoto
+---
+
+# Contributing to Resoto
```

Comment:
```suggestion
# Contributing to Resoto Components
```
Since we now have documentation that refers to them in this way: https://resoto.com/docs/concepts/components/

Reply:
(also update `sidebar_label` above)